File: Expresiones/Ejercicio3-Expresiones.cpp
//3. Escribe la siguiente expresión en C++

#include<iostream>
using namespace std;

int main(){
    float a, b, c, d, e, f, resultado = 0;

    cout<<"Ingrese valor de a: "; cin>>a;
    cout<<"Ingrese valor de b: "; cin>>b;
    cout<<"Ingrese valor de c: "; cin>>c;
    cout<<"Ingrese valor de d: "; cin>>d;
    cout<<"Ingrese valor de e: "; cin>>e;
    cout<<"Ingrese valor de f: "; cin>>f;

    resultado = (a+(b/c))/(d+(e/f));

    cout.precision(3);
    cout<<"\nEl resultado es: "<<resultado<<endl;
    return 0;
}

File: Comparativa Python/Ejercicio3-Expresiones.py
# 3. Escribe la siguiente expresión (a+(b/c))/(d+(e/f))

a = float(input('Digite valor de a: '))
b = float(input('Digite valor de b: '))
c = float(input('Digite valor de c: '))
d = float(input('Digite valor de d: '))
e = float(input('Digite valor de d: '))
f = float(input('Digite valor de f: '))

resultado = (a+(b/c))/(d+(e/f))

print('El resultado es: ', str(round(resultado, 2)))
